### DISCUSSION TITLE
Ursina is asking for arm64 during install

### DISCUSSION THREAD

**Discussion by Asterisk3** [pokepetter/ursina#668] [Q&A]
I'm a very new programmer interested in giving Ursina a try.

-pip3 install ursina worked fine, but running the demo code raised this error:
![Image 4-8-24 at 8 18 AM](https://github.com/pokepetter/ursina/assets/106040868/6a17e822-68d1-4581-9a2e-e0c4060d67a7)

Googling similar issues said I needed to uninstall, purge pip3's cache, and reinstall with the correct version.

I'm not that familiar with Github's layout, and could really use some pointers.

**Comment by mibino** [comment]
mac?
